packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -184,8 +184,13 @@ export async function runBuildAndSubmitAsync(
     return;
   }
 
+  const { accountName } = Object.values(buildCtxByPlatform)[0];
```

Comment:
What if someone wants to have different owners for android and ios build?

I know that it's a vary weird corner case, but whole point of this task is to fix some rare inconsistency, so we should not cause another.